app/(candidate)/dashboard/outreach/components/OutreachCreateCards.js

```diff
@@ -21,6 +21,13 @@ export const OUTREACH_OPTIONS = [
     type: OUTREACH_TYPES.text,
     requiresPro: true,
   },
+  {
+    title: 'P2P Text',
+    impact: IMPACTS_LEVELS.medium,
+    cost: 0.035,
+    type: OUTREACH_TYPES.p2p,
+    requiresPro: true,
+  },
```

Comment:
Just update the existing outreach card for texting, and change it's `type` to `OUTREACH_TYPES.p2p`.  We're replacing that type of outreach, not adding a new one.